### DISCUSSION TITLE
Run "/hip-tests/samples/2_Cookbook/15_static_library/device_functions/", it report "Missing definition for '_Z9square_mei'"

### DISCUSSION THREAD

**Discussion by KimXiao1119** [CHIP-SPV/chipStar#812] [Q&A]
Hi guys,

Please help take a look at this issue.
![Uploading square.png…]()

I traced the compilation process and found that there was no unbundling the libHipDevice.a.

Thanks!

**Comment by KimXiao1119** [comment]
![square](https://github.com/CHIP-SPV/chipStar/assets/5965823/b9071aee-112f-4c7d-ac49-973b0aa535d9)

**Reply by pvelesko** [reply]
Just now saw this - message. If still relevant please open an issue
